src/app-layout/__tests__/main.test.tsx

```diff
@@ -115,6 +115,51 @@ describe.each([
   });
 });
 
+describe.each([
+  [
+    'drawers',
+    {
+      findElement: (wrapper: AppLayoutWrapper) => wrapper.findActiveDrawer(),
+      findToggle: (wrapper: AppLayoutWrapper) => wrapper.findDrawersTriggers()[0],
+      findClose: (wrapper: AppLayoutWrapper) => wrapper.findActiveDrawerCloseButton(),
+    },
+  ],
+] as const)('%s', (name, { findToggle, findElement, findClose }) => {
+  test('property is controlled', () => {
```

Comment:
Which property are we testing here? Should we set a value for `activeDrawerId`?

Reply:
I updated the test to include the `activeDrawerId` and `rerender`. Let me know if it looks okay.

Reply:
Looks good!